plugins/base/src/main/resources/dokka/templates/includes/page_heads.ftl

```diff
@@ -0,0 +1,6 @@
+<#macro display>
+    <title>${pageName}</title>
+    <@template_cmd name="pathToRoot">
+    <link href="${pathToRoot}images/logo-icon.svg" rel="icon" type="image/svg">
+    </@template_cmd>
+</#macro>
```

Comment:
What do you think about renaming the file to something like `page_metadata.ftl` or `page_head_metadata.ftl`? I think having some words from the [html spec](https://www.w3schools.com/html/html_head.asp) would help in intuitive understanding of what this template does.

I'd say `page_heads.ftl` is 50/50 - you assume it's elements inside `<head>`, but not 100% sure and have to double check. Moreover, it conflicts with `header.ftl` a bit